### PR TITLE
Create SECURITY.md

### PR DESCRIPTION
**Pull request by fabiand**
**What this PR does / why we need it**:
Document our security policy.

**Special notes for your reviewer**:
For now we point to Red Hat's Security team, as we are a main contributor.

```release-note
NONE
```

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
@slintes Prow tagging still does not work for the KubeVirt repo :smiley:

**Comment by slintes** [comment]
> @slintes Prow tagging still does not work for the KubeVirt repo

meh.... I thought only merging isn't enabled yet, but approving works :wink:

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+It is important to include the following details:
+  - The projects and versions affected
+  - Detailed description of the vulnerability
+  - Information on known exploits
+
+Vulnerability information is extremely sensitive. Please encrypt all security
+vulnerability reports using our *PGP key*
+
+A member of the Red Hat Product Security Team will review your e-mail and
+contact you to to collaborate on resolving the issue. For more information on
```
Comment (awels) [inline SECURITY.md]: double to

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+secalert@redhat.com. For issues related to Red Hat Products, please visit
+https://access.redhat.com/security/team/contact/.
+
+It is important to include the following details:
+  - The projects and versions affected
+  - Detailed description of the vulnerability
+  - Information on known exploits
+
+Vulnerability information is extremely sensitive. Please encrypt all security
+vulnerability reports using our *PGP key*
```
Comment (awels) [inline SECURITY.md]: I would make PGP key a link

**Pull request merged**